Sources/Orbit/Components/Dialog.swift

```diff
@@ -6,91 +6,109 @@ import SwiftUI
 public struct Dialog: View {
 
     public static let shadowColor = Color.inkNormal
-    
+
     let illustration: Illustration.Image
     let title: String
     let description: String
-    let buttons: ButtonConfiguration
+    let style: Style
+    let buttonConfiguration: Buttons
 
     public var body: some View {
         VStack(alignment: .center, spacing: .medium) {
-            if illustration != .none {
-                Illustration(illustration, layout: .frame(maxHeight: 120))
-                    .padding(.top, .medium)
-            }
+            Illustration(illustration, layout: .frame(maxHeight: 120))
+                .padding(.top, .medium)
 
             VStack(alignment: .center, spacing: .xSmall) {
-                Heading(title, style: .title4, alignment: .center)
-
+                Heading(title, style: .title3, alignment: .center)
                 Text(description, color: .inkLight, alignment: .center)
             }
 
             VStack(alignment: .center, spacing: .xSmall) {
-                switch buttons {
-                    case .primary(let title, let style, let action):
-                        Button(title, style: style.primaryStyle, action: action)
-                    case .primaryAndSecondary(
-                        let primaryTitle, let secondaryTitle, let style, let primaryAction, let secondaryAction
-                    ):
-                        Button(primaryTitle, style: style.primaryStyle, action: primaryAction)
-                        Button(secondaryTitle, style: style.secondaryStyle, action: secondaryAction)
-                }
+                buttons
             }
         }
         .padding(.medium)
         .background(
             RoundedRectangle(cornerRadius: .small)
                 .fill(.white)
-                .shadow(color: Self.shadowColor.opacity(0.3), radius: .medium, x: 0, y: .medium / 2)
-        )
-        .overlay(
-            RoundedRectangle(cornerRadius: .small)
-                .strokeBorder(Self.shadowColor.opacity(0.15), lineWidth: BorderWidth.thin)
+                .shadow(color: Self.shadowColor.opacity(0.7), radius: .xxxLarge, x: 0, y: .xxxLarge / 2)
         )
+        .frame(maxWidth: Layout.readableMaxWidth / 2, maxHeight: .infinity)
+        .padding(.xLarge)
+        .background(Color.inkNormal.opacity(0.45))
+        .edgesIgnoringSafeArea(.all)
```

Comment:
Why are we ignoring safe area edges?

Reply:
Because this component is meant to always cover the whole screen modally (including tabbars etc)